src/components/frame/LoadingTemplate/LoadingTemplate.css

```diff
@@ -3,7 +3,7 @@
   position: absolute;
```

Comment:
I also would have liked to see this being moved over to tailwind, but we can tackle that another time.